{"name": "test_login_phone_error_case", "status": "passed", "attachments": [{"name": "log", "source": "508656b6-9039-4b99-9cf7-90e134292b8e-attachment.txt", "type": "text/plain"}], "start": 1605745876176, "stop": 1605745877735, "uuid": "53f6696f-0600-4b7e-a79d-d4a29438b144", "historyId": "210797769f28844d15197db9e5f676ed", "testCaseId": "7bcb06678aeec0849c5a6efc7c1a2546", "fullName": "case.login.test_login_case.TestLoginCase#test_login_phone_error_case", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "case.login"}, {"name": "suite", "value": "test_login_case"}, {"name": "subSuite", "value": "TestLoginCase"}, {"name": "host", "value": "FYYX-2019VXOSDQ"}, {"name": "thread", "value": "14712-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.login.test_login_case"}]}